.github/workflows/action-build.yml

```
@@ -26,22 +26,26 @@ jobs:
         run: |
           gh pr checkout ${{ github.event.pull_request.number }}
 
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+
       - uses: actions/setup-node@v3
         with:
           node-version: '16'
-          cache: 'yarn'
+          cache: 'pnpm'
 
-      - run: yarn install --frozen-lockfile
+      - run: pnpm install --frozen-lockfile
 
-      - run: yarn build
+      - run: pnpm build
 
       - name: Commit and push if needed
         run: |
-          git add .
-          if [ -z "$(git status --porcelain)" ]; then
-            echo "💾 no changes to dist/index.js"
+          if [ "$(git diff --ignore-space-at-eol | wc -l)" -eq "0" ]; then
+            echo "❎ no changes"
             exit 0
           fi
+          git add .
```

```suggestion
          git add dist/index.js
```
or change the commit and echo lines to say it might have added more than just this file.

---

No, I do not want to try and figure out which dist it is.

Cause this will host multiple actions.

So it would be `./packages/action/dist` or `./packages/autolabeler-action/dist` or even more with the planned `./packages/pr-label-checker/dist/`

reason for `./packages/autolabeler-action` is that `./packages/app` wants to reuse autolabeler so a `./packages/autolabeler` is needed.